src/main/java/io/rcktapp/api/handler/firehose/FirehoseDb.java

```diff
@@ -31,25 +38,29 @@ public class FirehoseDb extends Db
 
    AmazonKinesisFirehoseAsync firehoseClient = null;
 
+   public FirehoseDb() {
+      super();
+      setType("firehose");
+   }
+
    @Override
    public void bootstrapApi() throws Exception
    {
-      AmazonKinesisFirehose firehoseClient = getFirehoseClient();
+      List<Pair<String, String>> nameActuals = new ArrayList<>();
+      // our local streams
+      getFirehoseClient().listDeliveryStreams(new ListDeliveryStreamsRequest().withDeliveryStreamType("DirectPut")).getDeliveryStreamNames().forEach(name -> nameActuals.add(Pair.of(name.toLowerCase(), name.toLowerCase())));
+      // our defined aliases
+      Stream.of(includeStreams.split(",")).map(part -> part.split("\\|")).forEach(arr -> nameActuals.add(Pair.of(arr[0], arr.length > 1 ? arr[1] : arr[0])));
 
       this.setType("firehose");
 
       if (!J.empty(includeStreams))
       {
-         String[] parts = includeStreams.split(",");
-         for (String part : parts)
+         for (Pair<String, String> stream : nameActuals)
          {
-            String[] arr = part.split("\\|");
-            String collectionName = arr[0];
-            String streamName = collectionName;
-            if (arr.length > 1)
-            {
-               streamName = arr[1];
-            }
+            log.info("bootstrap {} stream {}", getType(), stream);
+            String collectionName = stream.getKey();
+            String streamName = stream.getValue();
```

Comment:
This logic is to use our (previously unused) firehose Client to retrieve the list of fireshoses available, and put them into the active api definition.  We don't have very many and its annoying to have to go back and reconfigure and redeploy snooze every single time we want to try working with a different firehose stream.

The 'includeStreams' has some optional alias names that can be defined.  Probably, we should make the references to those streams more configurable rather than making aliases, but backwards compatible.  We can do that going forward.